src/class-wpml-gutenberg-integration.php

```diff
@@ -193,7 +193,7 @@ private function render_inner_HTML( $block ) {
 			}
 
 		} else {
-			$content = $block->innerHTML;
+			$content = apply_filters( 'wpml_gutenberg_render_inner_HTML', $block->innerHTML, $block );
```

Comment:
@kagg-design There's already an existing filter that can be used instead of adding a new one. `wpml_update_strings_in_block`